template/setup/index.js

```diff
@@ -50,14 +82,6 @@ async function setup() {
   });
 
   console.log('🔄  Setting up...');
-  // add our update to package.json
-  const scripts = require('./scripts.json');
-  packageJson.scripts = { ...packageJson.scripts, ...scripts };
-  packageJson.jest = require('./jest.json');
-  packageJson['lint-staged'] = require('./lintStaged.json');
-
-  console.log('\n📝  Writing package.json...');
-  writeFile('../package.json', JSON.stringify(packageJson, null, 2));
```

Comment:
are these unnecessary now?

Reply:
Yes, because we added a package.json to the template, so we don't need to write one